### DISCUSSION TITLE
load_model function giving error - File name too long

### DISCUSSION THREAD

**Discussion by rahulsahai78** [pycaret/pycaret#2260] [General]
![image](https://user-images.githubusercontent.com/90679664/157451863-85543f59-3c75-4404-8bd0-29b9b6061498.png)
Getting this error on loading the model saved earlier by save_model function.
please help!

**Comment by moezali1** [comment]
You have to pass model name as a string in load_model. I see you have passed the variable name as oppose to model string.
